spec/models/denounce_spec.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: denounces
+#
+#  id          :bigint           not null, primary key
+#  anonymouse  :boolean          default(FALSE)
+#  area        :integer
+#  description :string
+#  event_date  :date
+#  title       :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  user_id     :integer
+#
```

Comment:
If all these comments are not necessary you can remove them, so that the code looks cleaner.

Reply:
> 
> 
> If all these comments are not necessary you can remove them, so that the code looks cleaner.

Hi @YilKanda !, Thanks for the feedback. We think this helps as documentation to improve development agility. We can make a refactoring branch to remove this when we finish the development